Drop py2 from wiki (traversal and zodb)

stevepiercy opened the pull request.

stevepiercy committed:
Remove Python versions mention

stevepiercy committed:
Correct grammar, use shorter sentences, rewrap to one sentence per line.

stevepiercy committed:
Correct grammar, use shorter sentences, rewrap to one sentence per line, inline links, update outputs.

stevepiercy committed:
remove extra whitespace

stevepiercy committed:
synch up source files with actual cookiecutter results

stevepiercy committed:
Add files from cookiecutter

stevepiercy committed:
Synch up src files

stevepiercy committed:
Clean up application configuration in __init__.py

stevepiercy committed:
Clean up models, now in a package

stevepiercy committed:
Clean up views, now in a package

stevepiercy committed:
Rewrite application configuration to mirror the URL-dispatch + SQLAlchemy wiki style

stevepiercy committed:
Clean up grammar in models section

stevepiercy committed:
Remove redundancy in appmaker narrative

stevepiercy committed:
Add notfound_view function description and 404.pt template.

stevepiercy committed:
Clean up configuration for development.ini

stevepiercy committed:
Clean up introduction and delete the database sections of definingmodels.rst

stevepiercy committed:
Synch up src/models files

stevepiercy committed:
Update models and view application sections

stevepiercy committed:
rewrap introduction, spell out contractions

stevepiercy committed:
Synch non-view and non-template src/views files

stevepiercy committed:
Clean up introduction to views, synch views files

stevepiercy committed:
Update view_wiki narrative

stevepiercy committed:
Add trailing slashes to intersphinx in hopes of fixing failed docs builds

stevepiercy committed:
Revise view_page narrative

stevepiercy committed:
Add a note for resources and traversal chapters.

stevepiercy committed:
Update models and view application sections

stevepiercy committed:
Use double backticks on objects

stevepiercy committed:
Revise the add_page view section narrative

stevepiercy committed:
Use proper reST syntax and directives

stevepiercy committed:
Rewrap edit_page view section

stevepiercy committed:
Clean up templates narrative.

stevepiercy committed:
Fix imports, contexts, paths to templates, and refactor `content` to `page_text` (there's too many "content" already and is confusing)

stevepiercy committed:
Delete obsolete views.py

stevepiercy committed:
Add import and return a Page object for the notfound view

stevepiercy committed:
Align line numbers with src in views

stevepiercy committed:
Fix up templates to inherit from layout.pt

stevepiercy committed:
Fix up template for display on docs

stevepiercy committed:
Fix up line numbers, add a not found error check

stevepiercy committed:
Add section for notfound

stevepiercy committed:
Rewrap intro and add dependencies to authorization

stevepiercy committed:
We're adding both authn and authz

stevepiercy committed:
rewrap users and groups section, shorter sentences, fix grammar

stevepiercy committed:
rewrap add an ACL section, shorter sentences, fix grammar

stevepiercy committed:
rewrap  policies and permissions sections, shorter sentences, fix grammar

stevepiercy committed:
indent literalincludes, add login.pt template and view

stevepiercy committed:
Use path for pagename, not subpath

stevepiercy committed:
Update theme.css to display text in inputs
- See #3431

stevepiercy committed:
rewrap review, shorter sentences, fix grammar.
align line numbers with code.

stevepiercy committed:
synch up src/authorization files

stevepiercy committed:
Fix template to include either the page name or a title for the title tag

stevepiercy committed:
Fix path to template

mmerickel commented:
@stevepiercy I rendered and walked through the tutorial and it's looking pretty good. This slipped through in 1.10 but we I want the `settings['tm.manager_hook'] = ...` removed from the zodb cookiecutter. There are bugs with it right now and zodb is more comfortable using the threadlocal transaction manager so I'd like to leave it that way. It's a one line change but I'm sure it'll affect the line numbers.

https://github.com/Pylons/pyramid-cookiecutter-starter/blob/40a29576147c9f52a9e5a96d3f994b301edc65e0/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/base___init__.py#L21

stevepiercy committed:
Remove settings for tm.manager_hook from __init__.py

stevepiercy committed:
Align line numbers with code
- See https://github.com/Pylons/pyramid-cookiecutter-starter/pull/66

stevepiercy commented:
Thanks. Removed and pushed commits.

stevepiercy committed:
Synch src files between auth and tests

stevepiercy committed:
rewrap narrative, adjust grammar

stevepiercy committed:
Add note about docutils deprecation warnings.
Submitted a PR to fix the warnings in WebTest https://github.com/Pylons/webtest/pull/207, which usually gets reviewed and merged rapidly, so didn't bother to include a warning for it.

stevepiercy committed:
Fix imports. Change info key from content to page_text.

stevepiercy committed:
Update distributing with terms, rewrap

stevepiercy commented:
All done. Ready for final review.
